sql/core/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -674,6 +674,12 @@ object SQLConf {
       .stringConf
       .createWithDefault(TimeZone.getDefault().getID())
 
+  val PARQUET_TABLE_INCLUDE_TIMEZONE =
+    buildConf("spark.sql.session.parquet.timeZone")
+      .doc("""Enables inclusion of parquet timezone property in newly created parquet tables""")
```

Comment:
There should be a config option for writing "UTC" to the table property when creating tables, not for writing the local timezone.